SPECS/maven/maven.spec

```diff
@@ -0,0 +1,241 @@
+%global debug_package %{nil}
+
+%define m2_cache_tarball_name apache-%{name}-%{version}-m2.tar.gz
+%define licenses_tarball_name apache-%{name}-%{version}-licenses.tar.gz
+
+%define offline_build -o
+
+%define _prefixmvn %{_var}/opt/apache-%{name}
+%define _bindirmvn %{_prefixmvn}/bin
+%define _libdirmvn %{_prefixmvn}/lib
+
+# maven 1.0 package version being used. This needs to be updated in case of updates in 1.0.
+%define mvn_1_0_pmc_ver 3.5.4-13
+
+Summary:        Apache Maven
+Name:           maven
+Version:        3.8.4
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/System
+URL:            https://maven.apache.org/
+Source0:        https://archive.apache.org/dist/%{name}/%{name}-3/%{version}/source/apache-%{name}-%{version}-src.tar.gz#/apache-%{name}-%{version}-src.tar.gz
+# @TODO
+# Since bootstrap has been removed for maven, it requires a pre-built maven binary to build itself.
+# Relying on 1.0 maven rpm to provide the mvn binary for the build.
+%ifarch x86_64
+Source1: https://cblmarinerstorage.blob.core.windows.net/sources/core/maven-3.5.4-13.cm1.x86_64.rpm
+%else
+Source1: https://cblmarinerstorage.blob.core.windows.net/sources/core/maven-3.5.4-13.cm1.aarch64.rpm
+%endif
+
+# CBL-Mariner build are without network connection. Hence, we need to generate build caches as tarballs to build
+# rpms in offline mode.
+# In order to generate tarballs, use "maven_build_caches.sh".
+# ./maven_build_caches.sh -v <Maven version string> -a <x86_64 | aarch64>
+# ex: ./maven_build_caches.sh -v 3.8.4 -a x86_64
+Source3:        %{m2_cache_tarball_name}
```

Comment:
Nit: I maybe renumber so there is a Source2?

Reply:
Done